Unit1/Module1.4/Mini-Project1.4/README.md

```diff
@@ -4,3 +4,6 @@
 
 ### Mini Project Instructions
 
+1a. For this mini-project, simply create a branch containing all of your coursework for Unit 1.4.
```

Comment:
Minor suggestion, if its not too much trouble. It would probably reduce confusion for people if we specified the list of code files/folders to turn in, or otherwise somehow made it a little more specific.